Not building on Ubuntu 20.04 with clang version 10.0.0-4ubuntu1

HI there,

I was trying to build this tool on Ubuntu 20.04 with Clang-10.0 but encountered this error:

```
>> bash make.sh 
code/src/utils.cpp:41:9: error: use of undeclared identifier 'getuid'
    if (getuid() != 0) {
        ^
code/src/utils.cpp:47:16: error: use of undeclared identifier 'getuid'
    if ((uid = getuid()) == 0) {
               ^
code/src/utils.cpp:57:16: error: use of undeclared identifier 'getgid'
    if ((gid = getgid()) == 0) {
               ^
code/src/utils.cpp:65:9: error: use of undeclared identifier 'setgid'
    if (setgid(gid) != 0) {
        ^
code/src/utils.cpp:69:9: error: use of undeclared identifier 'setuid'
    if (setuid(uid) != 0) {
        ^
code/src/utils.cpp:75:9: error: use of undeclared identifier 'setuid'
    if (setuid(0) == 0 || seteuid(0) == 0) {
        ^
code/src/utils.cpp:75:27: error: use of undeclared identifier 'seteuid'
    if (setuid(0) == 0 || seteuid(0) == 0) {
                          ^
7 errors generated.
```

Could you help me with it?

Anyway, is there any plan to make this tool available via `pip` or `conda` install? or maybe a static build? That would be really nice.

--- a/code/src/utils.cpp
+++ b/code/src/utils.cpp
@@ -1,6 +1,7 @@
 #include "utils.h"
 
 #include <stdint.h>
+#include <unistd.h>
 #include <ctime>
 #include <cstdarg>
 #include <cstdio>

